packages/@lwc/engine/dom/src/renderer.ts

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+import { hasOwnProperty, isUndefined } from '@lwc/shared';
+
+import { Renderer } from '../../src';
+
+// TODO [#0]: Evaluate how we can extract the `$shadowToken$` property name in a shared package
+// to avoid having to synchronize it between the different modules.
+export const syntheticShadow = hasOwnProperty.call(Element.prototype, '$shadowToken$');
```

Comment:
@ekashida This file also contains couple of unlinked todos, let's make sure we either track them as part of the SSR worksheet, or we create the appropriate issues to track them long term.

Reply:
I don't see anyone importing this named export, maybe it is just a thing on the renderer

Reply:
It is used from `isNodeFromTemplate`: https://github.com/salesforce/lwc/pull/1879/files#diff-b04e1c5f95306603cc39354f12ba9e19R10